iron-menu-behavior.html

```diff
@@ -361,6 +368,20 @@
     _activateHandler: function(event) {
       Polymer.IronSelectableBehavior._activateHandler.call(this, event);
       event.stopPropagation();
+    },
+
+    /**
+     * Updates this element's tab index when it's enabled/disabled.
+     * @param {boolean} disabled
+     */
+    _disabledChanged: function(disabled) {
+      if (this.getAttribute('tabindex') != null)
+        this._previousTabIndex = this.tabIndex;
```

Comment:
I've updated the code based on your feedback to be more clear and added a test based on a similar example to what you proposed.

`
// This is expected behavior.
this.setAttribute('tabindex', 5);
this.disabled = true;
this.removeAttribute('tabindex');  // This is a no-op b/c tab index is not set.
this.disabled = false;
// tabIndex is now 5. This may be unexpected for user, but is by design.
`

`
// Added test for this case:
this.setAttribute('tabindex', 5);
this.disabled = true;
this.setAttribute('tabindex', 51);
this.disabled = false;  // Tab index is preserved.
// tabIndex is still 51
`